packit_service_fedmsg/consumer.py

```diff
@@ -87,6 +87,7 @@ def fedora_messaging_callback(self, message: Message):
 
         logger.info(message.body.get("what"))
         message.body["topic"] = message.topic
+        message.body["timestamp"] = message.timestamp
```

Comment:
Okay, so my recommendation was not correct, should have tested first or make it obvious that it's just an assumption.

```python
message._headers.get("sent-at")
```
Some debugging shows that there is no `message.timestap`. But there is the above. `_headers` is not public, but could be [made so](https://github.com/fedora-infra/fedora-messaging/blob/master/fedora_messaging/message.py#L359).

Yesterday this seemed to be the most solid solution, but now it's getting complicated, so I don't know how to proceed...

1. fix this, with the above;
2. just record `utcnow()` at this point;
3. talk with Copr folks to include a timestamp in their start and end messages (on a long term of course).

Reply:
I vote for utcnow() - easiest and shouldn't break anything

Reply:
+1 to utcnow(). I am going to change it